lib/redcord/server_scripts/create_hash.erb.lua

```diff
@@ -29,10 +29,12 @@ local id, hash_tag = unpack(KEYS)
 local model, ttl = unpack(ARGV)
 local key = model .. ':id:' .. id
 
-local index_attr_pos = 5
+local index_attr_pos = 6
 local range_attr_pos = index_attr_pos + ARGV[3]
+local custom_attr_pos = range_attr_pos + ARGV[4]
```

Comment:
dont forget to put an example explaining the input format in the header!

Reply:
added